### PR TITLE
storage: use static metrics for KV_COMMAND_COUNTER

### PR DESCRIPTION
**Pull request by fredchenbj**
Signed-off-by: fredchenbj <cfworking@163.com>

## What have you changed? (mandatory)

use static metrics for "KV_COMMAND_COUNTER_VEC"

## What are the type of the changes? (mandatory)
- Improvement (change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

manual tests

## Does this PR affect documentation (docs) update? (mandatory)

no

## Does this PR affect tidb-ansible update? (mandatory)

no

### CODE REVIEW

File: src/storage/metrics.rs [hunk 1]
```diff
@@ -25,6 +25,30 @@ make_static_metric! {
...
+        key_mvcc,
+        start_ts_mvcc,
+        raw_put,
+        raw_batch_put,
+        raw_delete,
+        raw_delete_range,
+        raw_batch_delete,
+    }
+
+    pub struct KvCommandCounterVec: IntCounter {
```
Comment (breeswish) [inline src/storage/metrics.rs]: I believe that the enum of KvCommandCounterVec should be the same as SchedDurationVec?

Comment (fredchenbj) [reply src/storage/metrics.rs]: OK, I have change this.

**Comment by siddontang** [comment]
Thanks @fredchenbj 

Can you show us the benchmark results?

**Comment by fredchenbj** [comment]
> Thanks @fredchenbj
> 
> Can you show us the benchmark results?

@siddontang In my test case, i.e. command `./bin/go-ycsb load tikv -P workloads/workloada -p tikv.pd="10.103.18.194:2379" -p tikv.type="raw" -p tikv.conncount=128 --threads=32` uses `raw_put` to write 1000000 records, the ops is almost the same, only max latency is less (494829 -> 458178). So maybe only this pr works less obviously.

**Pull request merged**